### DISCUSSION TITLE
Hey i need an "ai pro" for my project on a real time ip cameras and microphones recorder and the ai highlite special voice recognition and movements

### DISCUSSION THREAD

**Discussion by sebspion** [community/community#36365] [Other Feature Feedback, Questions, & Ideas]
I really need a pro in  a.i. and voice recognition  . Ils m'y new project a big one but i need at least one worker better 2 . Answer fast and maybe i Will take you for my n'est projects.

**Comment by ossanna16** [comment]
Hi @sebspion,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on AI. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by tawfik-s** [comment]
I think one of the best two places to search is 
kaggle  and LinkedIn   you will be helped there
